tests/LucidFrame/Console/ConsoleTableTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+use LucidFrame\Console\ConsoleTable;
+
+class ConsoleTableTest extends PHPUnit_Framework_TestCase{
+
+	public function testBorderedTableDefault(){
+
+		$borderedTableDefault = "
++----------+------+
+| Language | Year |
++----------+------+
+| PHP      | 1994 |
+| C++      | 1983 |
+| C        | 1970 |
++----------+------+";
+
+		$table = new ConsoleTable();
+		$table
+		    ->addHeader('Language')
+		    ->addHeader('Year')
+		    ->addRow()
+		        ->addColumn('PHP')
+		        ->addColumn(1994)
+		    ->addRow()
+		        ->addColumn('C++')
+		        ->addColumn(1983)
+		    ->addRow()
+		        ->addColumn('C')
+		        ->addColumn(1970);
+
+		$this->assertEquals(trim($borderedTableDefault), trim($table->getTable()));
+	}
+
+	public function testBorderedTableWithPaddingWidth2(){
+
+		$borderedTableWithPaddingWidth2 = "
++------------+--------+
+|  Language  |  Year  |
++------------+--------+
+|  PHP       |  1994  |
+|  C++       |  1983  |
+|  C         |  1970  |
++------------+--------+";
+
+		$table = new ConsoleTable();
+		$table
+		    ->setHeaders(array('Language', 'Year'))
+		    ->addRow(array('PHP', 1994))
+		    ->addRow(array('C++', 1983))
+		    ->addRow(array('C', 1970))
+		    ->setPadding(2);
+
+		$this->assertEquals(trim($borderedTableWithPaddingWidth2), trim($table->getTable()));
+	}
+
+	public function testBorderedTableWithLeftMarginWidth4(){
+
+		$borderedTableWithLeftMarginWidth4 = "
+    +----------+------+
+    | Language | Year |
+    +----------+------+
+    | PHP      | 1994 |
+    | C++      | 1983 |
+    | C        | 1970 |
+    +----------+------+
+";
+
+
+		$table = new ConsoleTable();
+		$table
+		    ->setHeaders(array('Language', 'Year'))
+		    ->addRow(array('PHP', 1994))
+		    ->addRow(array('C++', 1983))
+		    ->addRow(array('C', 1970))
+		    ->setIndent(4);
+
+		$this->assertEquals($borderedTableWithLeftMarginWidth4, sprintf("\n%s", $table->getTable()));
+	}
+
+	public function testNonBorderedTableWithHeader(){
+
+		$nonBorderedTableWithHeader = "
+ Language  Year 
+----------------
+ PHP       1994 
+ C++       1983 
+ C         1970 ";
+
+		$table = new ConsoleTable();
+		$table
+		    ->setHeaders(array('Language', 'Year'))
+		    ->addRow(array('PHP', 1994))
+		    ->addRow(array('C++', 1983))
+		    ->addRow(array('C', 1970))
+		    ->hideBorder();
+
+		$this->assertEquals(trim($nonBorderedTableWithHeader), trim($table->getTable()));
+	}
+
+	public function testNonBorderedTableWithoutHeader(){
+
+		$nonBorderedTableWithoutHeader = "
+ PHP  1994 
+ C++  1983 
+ C    1970 ";
+
+		$table = new ConsoleTable();
+		$table
+		    ->addRow(array('PHP', 1994))
+		    ->addRow(array('C++', 1983))
+		    ->addRow(array('C', 1970))
+		    ->hideBorder();
+
+		$this->assertEquals(trim($nonBorderedTableWithoutHeader), trim($table->getTable()));
+	}
+}
```

Comment:
Can you please add a new blank line at the end of the file?
And this package follows the coding standard [PSR-2](http://www.php-fig.org/psr/psr-2/), so can you please correct it according to PSR-2, for example, 
- indentation with 4 spaces
- no mix of tabs and spaces in indentation
- opening braces for classes MUST go on the next line
- opening braces for methods MUST go on the next line
- no trailing spaces
and so on.